src/pages/Login.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import Input from "../components/Input";
+import logo from "../assets/Logo.svg";
+import googleLogo from "../assets/google.png";
+import naverLogo from "../assets/naver.png";
+import kakaoLogo from "../assets/kakao.png";
+
+function Login() {
+  const [emailInputValue, setEmailInputValue] = useState("");
+  const [passwordInputValue, setPasswordInputValue] = useState("");
+
+  return (
+    <div className="flex flex-col justify-center items-center gap-5">
+      <form
+        name="login"
+        className="flex flex-col justify-center items-center gap-5"
+      >
+        <img src={logo} alt="pchedule logo" className="w-[300px] scale-70" />
+        <div className="flex flex-col gap-1 justify-center items-start">
+          <h2 className="text-[13px]">이메일</h2>
+          <Input
+            type={"email"}
+            placeHolder={"이메일을 입력하세요."}
+            value={emailInputValue}
+            setValue={setEmailInputValue}
+            required={true}
+          />
+        </div>
+        <div className="flex flex-col gap-1 justify-center items-start">
+          <h2 className="text-[13px]">비밀번호</h2>
+          <Input
+            type={"password"}
+            placeHolder={"비밀번호를 입력하세요."}
+            value={passwordInputValue}
+            setValue={setPasswordInputValue}
+            required={true}
+          />
+        </div>
+        <button className="w-[300px] h-8 m-3 rounded-xl bg-[#D9D9D9] hover:bg-[#5AA5B2]">
+          Login
+        </button>
```

Comment:
아직 버튼 공통컴포넌트는 없는걸까요?

Reply:
버튼 컴포넌트가 있긴한데 취소랑 확인이랑 같이 묶여있어서 이건 따로 만들었어요- 하나짜리로 만들어서 사용해야 할까요-?!

Reply:
아하 보통 버튼 컴포넌트는 공통된 디자인만 사용하고 텍스트나 기능은 props로 받는 형태로 만듭니다!

예시코드)

```suggestion
       <button
      type="button"
      onClick={onClick}
      disabled={disabled}
      className={`w-[300px] h-8 m-3 rounded-xl bg-[#D9D9D9] hover:bg-[#5AA5B2] ${className}`}
    >
      {children}
    </button>
```

Reply:
아하 ㅎㅎㅎ 저희 여기 사용되는 버튼이 디자인이 좀 달라서 여기에서 만들어서 사용하기로 했습니다!!